### PR TITLE
Windows: use Winpcap, drop DNET dependency

### PR DESCRIPTION
**Pull request by p-l-**
This is [bitbucket pull request #158](https://bitbucket.org/secdev/scapy/pull-requests/158), on behalf on @n1nj4sec.

It is augmented to support Windows XP, without the need of PowerShell.

**It is not ready for merge** and needs a lot of tests.

### CODE REVIEW

**Comment by p-l-** [comment]
@n1nj4sec, I have probably broken support for hosts with PowerShell installed. Mind testing? I'll test also as soon as I can.

**Comment by n1nj4sec** [comment]
@p-l- I sent you a PR with some modifications

**Comment by p-l-** [comment]
@guedou tested and OK (ready to merge) for me.

**Comment by n1nj4sec** [comment]
awesome ! it seems good to me too.

File: scapy/arch/pcapdnet.py [hunk 1]
```diff
@@ -16,9 +16,307 @@
 from scapy.supersocket import SuperSocket
 from scapy.error import Scapy_Exception
 import scapy.arch
+import socket
 
 
+if conf.use_winpcapy:
+  #mostly code from https://github.com/phaethon/scapy translated to python2.X
```
Comment (guedou) [inline scapy/arch/pcapdnet.py]: I think that it is better to give the full link to the commit/file that was used.

**Comment by guedou** [comment]
It could test this code on Windows. I tried with Amazon EC2 but could not make it work. So, I will leave you guys with this responsibility =)

The code looks nice, however it could be nice to make it more compliant to PEP-08 using pylint or flake8

**Comment by hsluoyz** [comment]
Hi.

You would like to try Npcap: https://github.com/nmap/npcap.

It's another fork of WinPcap with multiple new features.

**Pull request merged**